### PR TITLE
Make navbar sticky with conditional styling enhancements

### PR DESCRIPTION
**Pull request by Aman-G-upta**
This pull request makes the navbar sticky and adds dynamic style adjustments for smoother UX during scrolling. Changes include:
Changes made 
- Maked navbar sticky 
- Enhanced visual transition for better user experience.

Please merge it @AlfiyaSiddique

### CODE REVIEW

**Comment by AlfiyaSiddique** [comment]
@Aman-G-upta I have asked you to go through #269 for detailed description but I think you have missed that.
The feature behavior should be as follows:
 **make the navbar sticky make sure it becomes sticky only after the first 100vh**
 
 Please fix it, you can refer this site for the navbar sticky behaviour expected
https://www.pexels.com/

**Comment by Aman-G-upta** [comment]
Maked changes as requested navbar will appear after 100vh 
Please merge it @AlfiyaSiddique

**Comment by AlfiyaSiddique** [comment]
@Aman-G-upta the build is failing: It says the scroll progress bar file is not found, please fix it asap
image

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -24,6 +25,18 @@ import ForgotPassword from "./Pages/ForgotPassword.jsx";
...
+//       {/* Render ScrollProgressBar at the top */}
+//       <ScrollProgressBar />
+//       <Navbar />
+//       {/* Other components, routes, or layouts */}
+//     </div>
+//   );
+// };
+
 function App() {
   const [showScroll, setShowScroll] = useState(false);
```
Comment (AlfiyaSiddique) [inline frontend/src/App.jsx:41]: Remove this comment @Aman-G-upta

File: frontend/src/Components/Navbar.jsx [hunk 2]
```diff
@@ -46,7 +46,9 @@ const Navbar = () => {
 
   useEffect(() => {
     const handleScroll = () => {
+      console.log(window.scrollY);
       if (window.scrollY > window.innerHeight) {
```
Comment (AlfiyaSiddique) [inline frontend/src/Components/Navbar.jsx:50]: @Aman-G-upta remove this log

File: frontend/src/Components/Navbar.jsx [hunk 3]
```diff
@@ -68,10 +70,21 @@ const Navbar = () => {
   };
 
   return (
+    // <nav
+    //   className={`z-50 relative navbar ${isSticky ? "fixed top-0 left-0 w-full bg-white shadow-md" : ""
+    //     }`}
+    // >
+    
+
```
Comment (AlfiyaSiddique) [inline frontend/src/Components/Navbar.jsx:78]: @Aman-G-upta remove all unnecessary comment

**Comment by AlfiyaSiddique** [comment]
@Aman-G-upta, I have fixed the deployment issue. There are some minor comments above. Please fix them as soon as possible so that I can merge your PR.
Also, make sure your PR has latest updates from the master branch

**Review by AlfiyaSiddique — APPROVED** [review]

**Pull request merged**